passes/cmds/select.cc

```diff
@@ -1099,6 +1113,9 @@ struct SelectPass : public Pass {
 		log("    <obj_pattern>\n");
 		log("        select the specified object(s) from the current module\n");
 		log("\n");
+		log("Prefix the following patterns with '=' if the pattern should match black-/\n");
+		log("white-box modules and their contents.\n");
```

Comment:
```suggestion
		log("By default, <[mod|obj]_pattern> will not match black/white-box modules or");
		log("their contents. To include such modules, prefix these patterns with '='.\n");
```

Reply:
`<[mod|obj]_pattern>` are components of a pattern. `=` is a prefix to the entire pattern, not to either component.

Reply:
```suggestion
		log("By default, patterns will not match black/white-box modules or their");
		log("contents. To include such objects, prefix the pattern with '='.\n");
```

Reply:
If that doesn't do it let's stop playing whack a mole. Just edit my suggestion and commit it.